File: Python/PRAK604-2210817210005-MFithra_Siroja.py
a=0;b=0
asli = input()
palsu = input()
kode = len(asli)
pesan = len(palsu)
if kode == pesan :
    for i in range(kode):
        if(asli[i]==palsu[i]):
            if (asli[i] == ' '):
                print(" ", end ='')
            else:
                print("*", end ='')
                a += 1
        else :
            print("#",end='')
            b += 1
    print("\n* = ",a)
    print("# = ",b)
    if(a >= b):
        print("Pesan Asli")
    else: 
        print("Pesan Palsu")
else:
 print("panjang Kalimat Berbeda, Pesan Palsu")

File: C/PRAK604-2210817210005-MFithra_Siroja.c
#include<stdio.h>
#include<string.h>

int main(void){
    char asli[100];
    char palsu[100];
    scanf("%[^\n]%*c", &asli);
    scanf("%[^\n]%*c", &palsu);
    int kode, pesan, n, a=0, b=0;
    kode= strlen(asli);
    pesan= strlen(palsu);
    if(kode != pesan){
        printf("\nPanjang kalimat berbeda, pesan palsu");}
    else{
        for (n=0; n<kode; n++){
            if(asli[n]==palsu[n]){
                if(asli[n]==' '){
                    printf(" "); }
                else{
                    printf("*");
                    a++; } }
            else{
                printf("#");
                b++; }
            }
        printf("\n*=%d", a);
        printf("\n#=%d", b);
        if(a>=b){
            printf("\nPesan Asli"); }
        else{
            printf("\nPesan Palsu"); }
            return 0; }
}
